### ISSUE TITLE
Security Suggestion

### ISSUE THREAD

**Issue by grullosgo** [zhukov/webogram#1932]
It is not an issue, but a suggestion.
I see with great concern, people constantly scammed on this platform. I think that an item could be added in the security settings so that the user can choose whether or not to receive private messages and from whom (as can be seen in other options), for example "all", "my contacts", "none" and default "my contacts". I am sure that with this simple option, thousands of scammers who have a big business on this platform would run out of business. Thanks for your attention.

**Comment by oxr463** [comment]
I don't think this is something that can be done here, since this is just a client. Otherwise, if you logged into Telegram Desktop or Mobile, you would still have all of those private messages.
